commit e34822145c9044013dbd2aaa25a1046433f2155e
Author: bertin <macportsuser@macports.org>
Date:   Fri Oct 10 19:10:41 2025 +0200

    /opt/local/linux-ports/audio/audacity/files/3.2.5/patch-fix-owner_less.diff

diff --git a/libraries/lib-realtime-effects/RealtimeEffectState.cpp b/libraries/lib-realtime-effects/RealtimeEffectState.cpp
index baa6001d0..2faa45438 100644
--- a/libraries/lib-realtime-effects/RealtimeEffectState.cpp
+++ b/libraries/lib-realtime-effects/RealtimeEffectState.cpp
@@ -304,7 +304,7 @@ struct RealtimeEffectState::Access final : EffectSettingsAccess {
       if (auto pOther = dynamic_cast<const Access*>(&other)) {
          auto &mine = mwState;
          auto &theirs = pOther->mwState;
-         auto less = std::owner_less{};
+         auto less = std::owner_less<>{};
          return !(less(mine, theirs) || less(theirs, mine));
       }
       return false;
